ci/gpu/build.sh

```diff
@@ -180,6 +180,7 @@ else
 
     gpuci_logger "Installing $CUDF_CONDA_FILE & $KAFKA_CONDA_FILE"
     conda install -c ${CONDA_ARTIFACT_PATH} "$CUDF_CONDA_FILE" "$KAFKA_CONDA_FILE"
+    conda install -c conda-forge transformers
```

Comment:
Installed temporarily for testing.

Reply:
@galipremsagar , What will be the recommended way to add such a testing-only dependency?

Reply:
If this is testing only, these are the places we will have to add `transformers` to:

https://github.com/rapidsai/integration/blob/branch-0.19/conda/recipes/rapids-build-env/meta.yaml

https://github.com/rapidsai/cudf/tree/branch-0.20/conda/environments (in all yml files)

and a new entry here in `setup.py` : https://github.com/rapidsai/cudf/blob/025c56ab1c4046f0f3d1d3ec768b1a69755643c8/python/cudf/setup.py#L35-L44

Reply:
Thanks for the help Prem, will add it at the needed places.

Reply:
Have added to setup.py with commit :  https://github.com/rapidsai/cudf/pull/7942/commits/6d3e86c664cb0df4d5913ec01d12afd2c76e3b13  and to conda files  with https://github.com/rapidsai/cudf/pull/7942/commits/4679c859e23b51f37de159850a2037fb4b48d335 .  


Will remove the dependency here as soon as integration PR  https://github.com/rapidsai/integration/pull/249 lands .